Dockerfile

```diff
@@ -1,62 +1,40 @@
-FROM quay.io/3scale/docker:ci-2.3.1-3
+FROM quay.io/3scale/system-builder:ruby24
 
 ARG SPHINX_VERSION=2.2.11
-ARG BUNDLER_VERSION=1.12.5
+ARG BUNDLER_VERSION=1.17.3
 ARG DB=mysql
-
-# Don't use ubuntu mirrors. Rather slow download, than failing build.
-RUN echo "deb http://archive.ubuntu.com/ubuntu precise main restricted universe multiverse\n\
-deb http://archive.ubuntu.com/ubuntu precise-updates main restricted universe multiverse\n\
-deb http://archive.ubuntu.com/ubuntu precise-backports main restricted universe multiverse\n\
-deb http://archive.ubuntu.com/ubuntu precise-security main restricted universe multiverse" > /etc/apt/sources.list
-
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 60C317803A41BA51845E371A1E9377A2BA9EF27F \
- && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D101F7899D41F3C3 \
- && apt-get update -y && apt-get install -y apt-transport-https \
- && echo 'deb http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu precise main' > /etc/apt/sources.list.d/toolchain.list \
- && curl -sL https://deb.nodesource.com/setup_6.x | bash - \
- && apt-install g++-4.8 nodejs squid3 libaio1 \
- && gem install bundler --version ${BUNDLER_VERSION} --no-document \
- && sed --in-place "s/databases 16/databases 32/" /etc/redis/redis.conf \
- && echo 'dns_nameservers 8.8.8.8 8.8.4.4' >> /etc/squid3/squid.conf \
- && cd /tmp && curl -o sphinxsearch.deb -J -L -O https://github.com/sphinxsearch/sphinx/releases/download/${SPHINX_VERSION}-release/sphinxsearch_${SPHINX_VERSION}-release-1.precise_amd64.deb \
- && curl -o unixODBC.deb -J -L -O https://github.com/3scale/unixODBC/releases/download/2.3.6-ubuntu-12.04/unixODBC_2.3.6_amd64.deb \
- && apt-install libodbc1 \
- && dpkg --install sphinxsearch.deb \
- && dpkg --install unixODBC.deb \
- && apt-get autoremove -y \
- && rm -f sphinxsearch_${SPHINX_VERSION}.deb unixODBC.deb
-
-ENV PATH="./node_modules/.bin:$PATH:/usr/local/nginx/sbin/" \
+ARG MASTER_PASSWORD=p
+ARG USER_PASSWORD=p
+
+ENV DISABLE_SPRING="true" \
+    ORACLE_SYSTEM_PASSWORD="threescalepass" \
+    NLS_LANG="AMERICAN_AMERICA.UTF8" \
+    TZ="UTC" \
+    MASTER_PASSWORD="${MASTER_PASSWORD}" \
+    USER_PASSWORD="${USER_PASSWORD}" \
+    LC_ALL="en_US.UTF-8" \
+    PATH="./node_modules/.bin:$PATH:/usr/local/nginx/sbin/" \
```

Comment:
I wonder why we need nginx?

Reply:
💯 .. I think it is fine to remove.. will do it